sdk/helper/certutil/helpers.go

```diff
@@ -536,7 +544,7 @@ func StringToOid(in string) (asn1.ObjectIdentifier, error) {
 // Validates that the combination of keyType, keyBits, and hashBits are
 // valid together; replaces individual calls to ValidateSignatureLength and
 // ValidateKeyTypeLength.
-func ValidateKeyTypeSignatureLength(keyType string, keyBits int, hashBits *int) error {
+func ValidateKeyTypeSignatureLength(keyType string, keyBits *int, hashBits *int) error {
```

Comment:
Bit concerned that it's surprising that a Validation function actually changes it's inputs.  Rather than use a pointer, I'd prefer a (named?) multi-return that returns the new keyBits/hashBits, and change the function name in some way to show that it's both validating and updating.

Reply:
Ah sorry, that snuck in from the earlier PR on signature bits, guess I brought in a C-ism. :dagger: 

How does multi-return work at the call site? Would we need to do:

```
var err error
...

if role.KeyBits, role.SignatureBits, err := certutil.ValidateKeyTypeSignatureLength(role.KeyType, role.KeyBits, role.SignatureBits); err != nil {
    return err
}
```

That gets really long and length, in my mind. I think the `&...` (for a C programmer) is sufficient warning to tell them to look at what the function is doing more closely.